browser/resources/settings/brave_routes.ts

```diff
@@ -71,6 +71,11 @@ export default function addBraveRoutes(r: Partial<SettingsRoutes>) {
   }
   if (pageVisibility.content) {
     r.BRAVE_CONTENT = r.BASIC.createSection('/braveContent', 'content')
+    // Move fonts from APPEARANCE to BRAVE_CONTENT
+    if (r.FONTS) {
+        delete r.FONTS
+    }
+    r.FONTS = r.BRAVE_CONTENT.createChild('/fonts');
```

Comment:
How about having `brave://settings/braveContent/fonts` instead of `brave://settings/fonts`?
```suggestion
    r.FONTS = r.BRAVE_CONTENT.createChild('/braveContent/fonts');
```

Reply:
I personally have no objections. The only thing I wonder if people who are used to going to `brave://settings/fonts` would be unhappy with the URL change.

Reply:
ah, i see. Using current one would be helpful for many users.